server/utils/migrate.go

```diff
@@ -0,0 +1,195 @@
+//from https://raw.githubusercontent.com/appleboy/mgo-migrate/master/migrate.go had to replace imports
+package utils
```

Comment:
Maybe this should be in its own `migrate` package? And could be under an `internal/` too so we aren't exporting it outside the repo.

Reply:
Would be good to reference the exact version copied too, instead of `master`.